content/working_with/console/templates-mgmt.md

```diff
@@ -50,29 +51,33 @@ In order to create new templates, sys-admins can choose the **Template Managemen
 Choosing this option will present all the existing templates and pages on the current manager. From here, you can create new templates and pages and edit existing ones.  
 
 
-### Creating new templates
+### Creating new template
```

Comment:
Since there is a change to singular - shouldn't it be 'Creating _**a**_ new template'?

Reply:
👍